libraries/platforms/hammerparrot-vcs/software/include/argp.h

```diff
@@ -0,0 +1,278 @@
+/* 
```

Comment:
LICENSE?

Reply:
for all copied files

Reply:
or point to them in the toolchain rather than copy

Reply:
Added the LICENSE information. For endian.h and features.h, I just used pointers to the original files. For the argp header, I put a pointer in the file as a comment. It comes from the linux newlib so there is no equivalent in the actual newlib+dramfs that we use.